package.json

```diff
@@ -17,7 +18,8 @@
     }
   },
   "scripts": {
-    "test": "./node_modules/.bin/standard && ./node_modules/.bin/mocha test"
```

Comment:
the command you had ran the standard command globally (on all files) and then ran the mocha runner on the /test path. I added the lint command so you can run "npm run lint filename.[extension]" for whatever specific file we would like to lint. I also made it so that npm run test does two specific things in regards to what "test" means -- 1) lint the contents of the /test path 2) run the mocha test suite on the /test path, that way you lint your new tests and run the test.